### DISCUSSION TITLE
Robby Russell - ohmyzsh

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#318] [Announcements]
- https://ohmyz.sh/
- https://github.com/ohmyzsh/ohmyzsh
- https://github.com/robbyrussell
- https://x.com/robbyrussell?lang=en
- https://robbyonrails.com/
- https://www.linkedin.com/in/robbyrussell/
- https://www.planetargon.com/

### ohmyzsh

- [x] What first got you into coding? Where did it lead you?
- [x] What is ohmyzsh? Where did it start? Why not choose fish or another shell?
- [x] What do you think lead to the growth of the project?
- [x] Whats the hardest part about building things for the terminal?
- [x] What are some of the most creative plugins you've seen? Favorite you use?
- [x] Do you have anything special about your theme? I've always dreamed of a star trek terminal, however unusable that might actually be
- [x] After maintaining a popular open source project for 15 years (and counting) what are some takeaways you have for making that possible? How have you not burned out?

### Ruby

- [x] What is planet argon?
- [x] What are some common trends you see among these apps you are contracted to work on?
- [x] In your eyes what's the key to creating maintainable and scalable software?
- [x] As a ruby dev what are your views on modern web dev outside of ruby?

### Future

- [x] ~What do you think the future of shells looks like? Terminals emulators are as old as computers themselves and we're at the dawn of a new age of computing.~

**Comment by robbyrussell** [comment]
Small note: last name has two Ls :-)
